src/bubble_swarm/angle_area.rs

```diff
@@ -0,0 +1,31 @@
+use crate::math2d::regioned_angle::{RegionedAngle, RegionedAngleTrait};
+
+#[derive(Debug, Clone, Copy, PartialEq, PartialOrd)]
+pub struct AngleArea {
```

Comment:
I am not happy with the name AngleArea anymore. Maybe we should just call it "Segment"